### ISSUE TITLE
API Feature Request: Legislator Vote Activity

### ISSUE THREAD

**Issue by sudolibre** [openstates/meta#23]
Problem: I would like to keep up to date on how my legislators are voting on bills.

Possible Solution: an api to GET up-to-date vote history for any single legislator.
Possible Solution 2: the ability to subscribe for legislators and get streaming notifications when new votes are available

There could potentially be an additional endpoint to find out the date the senate or house last voted. So it would be a two step process. Has the chamber voted since I last checked? If so check votes for my house and senate representative to see what's new.

**Comment by estaub** [comment]
You can get close, by requesting a [bill search](http://docs.openstates.org/en/latest/api/bills.html#methods) on `updated_since`, `state`, and maybe `chamber`.  The results are relatively small, since they have no votes.  Then do a bill detail on each result to get the votes.

**Comment by sudolibre** [comment]
Thanks for the comment @estaub! That's actually what I'm doing currently. I added the fields=votes parameter to the bill search. It doesn't give you the votes but it will give you an "id" field if the bill has been voted on. So I search for votes since the last time I checked, filter out the ones without an id field in the votes section, and then find the vote by doing a bill detail on each one and looking in the voter description for a string that matches the legislators last name + district as ordinal number (in GA we don't get leg_id). It works fine I just figured I'd throw this out if a new API is being designed.
